_posts/2022-08-24-kubeflow-1.6-release.md

```diff
@@ -0,0 +1,145 @@
+ [survey results](https://docs.google.com/document/d/1TLl8iFWQ-x2pVAHV9G8GFYMgAOuN2VFOnZV2JQNA-uc/edit#). Of the many
+ highlights, we would like to identify the growing request for model monitoring as shown in the chart below.
+
+![2022 User Survey Gaps In ML Activities and Workflow](/images/2022-08-24-kubeflow-1.6-release/2022-user-survey-gaps-in-ml-activities-and-workflow.png)
+
+We believe the increased focus on monitoring identifies Kubeflow’s maturing user base. The Community is making efforts
+ to explain Kubeflow’s current functionality and build into the users’ new monitoring requirements. In addition,
+ the user survey results and Kubeflow Working Group roadmaps will be discussed in the upcoming Kubeflow Summit. You can
+ learn more about the Kubeflow Summit [here](https://docs.google.com/document/d/1fLg1EqVtJekcXxT8GM_Rqg0-G-vIDVCrzuxE06Oq9dI/edit).
+
+To round back to the current software delivery, the following takes a deeper look at the 1.6 highlights in more detail.
+
+## Kubeflow v1.6 details
+
+From a new feature introduction standpoint, Kubeflow v1.6 includes an experimental release of the KFP v2’s new front-end,
+ back-end and SDK, which deliver a modern UI and DAG, first class support for metadata, and a simplified component
+ authoring experience. This slide deck, [Understand KFP v2](https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_20),
+ provides a good overview of v2. 
+
+<br />
+
+The KFP v2 alpha introduces: 
+
+- An Argo-agnostic approach for creating and running pipelines 
+- A brand new DAG visualization, which uses the Pipeline Template and MLMD in this [pipeline spec](https://github.com/kubeflow/pipelines/blob/master/api/v2alpha1/pipeline_spec.proto)
+- [Streamlined Component Authoring](https://docs.google.com/presentation/d/1HzMwtI2QN67xQp2lSxmuXhitEsukLB7mvZx4KAPub3A/edit#slide=id.gecbd775238_0_438)
+
+<br />
+
+You can learn more about KFP v2 and the related breaking changes in these docs:
+
+- [KFP SDK V2 CLI design](ahttps://docs.google.com/document/d/1Z79ebKaf3hjyadSRsme1Qo7F-Tqu4qM1cyKuLeu2PgA/edit)
+- [The SDK reference](https://kubeflow-pipelines.readthedocs.io/en/master/)
+- [Breaking changes](https://docs.google.com/document/d/1nCUUVRXexXbQ0LDkGHsMIBDSu1WvJA9Upy1JzybNVMk/edit) including an SDK change from `kfp.v2` to `kfp` 
+
+<br />
+
+For those users who are still relying on the KFP v1 functionality, Kubeflow v1.6 and its KFP v2 component are fully
+ tested and supported with the mature features in the legacy Kubeflow Pipelines v1.8 SDK. This provides the same
+ functionality that thousands of KFP v1 users leverage in production today.   
+
+<br />
+
+In addition to the KFP v2 alpha, Kubeflow 1.6 includes feature enhancements and operational improvements for Katib
+ users. These include support for:
+
+- The population based training algorithm, URL
```

Comment:
URL ?

Reply:
In the original google docs, it looks like the word "URL" was added to propose adding a link to the paper here. I removed the word "URL" and if you want to provide a paper that you want to link, I can link it here